packages/yaml/rollup.config.js

```diff
@@ -1,26 +1,10 @@
-import babel from 'rollup-plugin-babel';
-
 import pkg from './package.json';
 
 export default {
   input: 'src/index.js',
-  plugins: [
```

Comment:
what's the reason for removing the babel plugin?

Reply:
As we now have a Node 10 baseline, I do not think it makes sense to explicitly transpile down to loader compatibility targets as it both bloats files and makes them less readable. You may have hit an important point here, though: Technically, this would make this a breaking change as we would lift the compatibility baseline to Node 10. I would still love to keep this change but add a "BREAKING CHANGE" note to the commits if you agree. No need to release a new versions right away, though, IMO.

Reply:
Please feel free to update the engine to v10 in package.json on a new commit for the plugin (and any others that were changed with this) as a breaking change. Totally worth a major, and we've been kicking the can down the road on doing that.

Reply:
I added the engines property into the original commits and added them as breaking changes to the commit messages